lib/rspec/rabl/attribute_matcher.rb

```diff
@@ -11,7 +11,7 @@ def description
       end
 
       def failure_message
-        "expected #{parsed} to render #{rendered_attribute} = #{expected_value}"
+        "expected #{expected_value.inspect} in #{attribute_path}\n  got #{rendered_value.inspect}"
```

Comment:
Actually, maybe printing with `inspect` here isn't super great. It looks great for some things, but `DateTime` will look a little funky.